test/e2e/quadlet/env-file.container

```diff
@@ -0,0 +1,9 @@
+## assert-podman-final-args localhost/imagename
+## assert-podman-args --env-file /opt/env/abs-1 --env-file /opt/env/abs-2
+## assert-podman-args-regex --env-file rel-1
```

Comment:
Shouldn't this have an actual regexp for ther rel-1 absolute path?

Reply:
Like /.*/rel-1 or something?

Reply:
You're right. I've made it even more specific to make sure that the translation to absolute is to where the quadlet unit file is